### DISCUSSION TITLE
[Question] Update from 1.2.167 to 1.9.9

### DISCUSSION THREAD

**Discussion by VaclavElias** [amerkoleci/Vortice.Vulkan#56] [General]
Hi,

It seems that your Releases page https://github.com/amerkoleci/Vortice.Vulkan/releases is shy at the moment regarding the updates 🤣.

If Stride project would like to update from 1.2.167 to 1.9.9, do we need to be aware of something, any breaking changes.

Thank you.

**Comment by amerkoleci** [comment]
Hi,
I stopped using releases since long time, there shouldn't be any hard issues moving forward, it's current state is updated with vulkan sdk releases, feel free to ask if you have any issues or trouble during the road.

Cheers, 
Amer

**Reply by VaclavElias** [reply]
Thank you for a quick reply.

**Reply by VaclavElias** [reply]
We seem to have some issue, could you have a look at this? https://github.com/amerkoleci/Vortice.Vulkan/issues/58. Thank you.

**Reply by amerkoleci** [reply]
> We seem to have some issue, could you have a look at this? #58. Thank you.

Saw that, still need a repro case dude, I will gladly look at it.

Thanks,
Amer

**Reply by VaclavElias** [reply]
Thanks for a quick reply! [Feralnex](https://github.com/Feralnex) will provide that 🙂!
